### DISCUSSION TITLE
Computational Complexity by FLOPs and Number of Parameters

### DISCUSSION THREAD

**Discussion by YingdiZhang** [Project-MONAI/MONAI#4242] [Q&A] [answered]
Thanks for the great job.
I wonder if I could calculate the Computational Complexity by FLOPs and Number of Parameters of networks in MONAI， such as UNETR，3D Unet or other networks implement with MONAI blocks by myself .

**Comment by wyli** [comment, marked as answer]
Python libraries such as https://github.com/TylerYep/torchinfo prints some summaries.
